### DISCUSSION TITLE
Tool for writing papers?

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1114] [Q&A] [answered]
Hi all!

I'm starting on my draft for #pubathon2k21 and was wondering what tools are generally used for writing papers! I've heard a lot of talk about LaTeX via classes - is that preferred over something like Google Docs or MS Word?

**Comment by kubu4** [comment, marked as answer]
I'd stick with @yaaminiv's advice and just use what's most comfortable. Getting words down is the most important part. LaTeX is going to be overkill for the stuff we write about. 

I'd also recommend spending some time trying to use R Markdown/Bookdown to write your paper. Seems like it could really simplify updating/number figures and refs through the paper. Here's link to Slack thread, along with links to a presentation and GitHub repo which cover this topic: https://genefish.slack.com/archives/CPVU33TMM/p1605641861001100
